themes/bootstrap3/templates/Recommend/TopFacets.phtml

```diff
@@ -53,9 +53,9 @@
 <?php endif; ?>
 <?php
 $script = <<<JS
-VuFind.truncate.initTruncate('.top-facets-contents');
-// Close with JS (ensures content is shown to non-JS users)
-$(this).find('.collapse').removeClass('in');
-JS;
+                      VuFind.truncate.initTruncate('.top-facets-contents');
```

Comment:
This one seems a bit odd. Any idea what's going on here?

Reply:
Fixed.